src/main/java/com/tournament/cup/settings/TournamentClass.java

```diff
@@ -0,0 +1,190 @@
+package com.tournament.cup.settings;
+
+import com.tournament.cup.play.*;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Service;
+
+import java.util.Scanner;
+
+@Service
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TournamentClass {
```

Comment:
See the comments above. All the tournament configuration have same logic. No need to duplicate this logic based on tournament size